core/lib/dal/migrations/20240805143000_drop_picked_by_column.down.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE tee_verifier_input_producer_jobs ADD COLUMN picked_by TEXT;
```

Comment:
JFYI: migrations are a tool to change/rollback the database, so generally a single migration per PR is a better approach. Otherwise, if we have to revert PR, we also have to revert 2 migrations, which is unexpected.

Reply:
[Fixed](https://github.com/matter-labs/zksync-era/pull/2474/commits/caea7a5ccd7a86adcffd4a044b8dd7a1e27ec3d0). ✅